### PR TITLE
Felinids can actually eat mice and don't gross out

### PR DESCRIPTION
**Pull request by Kush1Push1**
## About The Pull Request

Removed "RAW" flag from dead mice.
Since humans still dislike "GORE" as well as some other races, it shouldn't cause issues.

## Why It's Good For The Game

A fix for #71138

## Changelog

:cl: SishTis
qol: Felinids can eat dead mice and don't gross out.
/:cl:

### CODE REVIEW

**Comment by Ghilker** [comment]
this makes no sense, it is raw meat, felinid code should be the one doing the proper checks

**Review by Fikou — CHANGES_REQUESTED** [review]
its a raw mouse, this is not an acceptable solution

**Comment by Kush1Push1** [comment]
> this makes no sense, it is raw meat, felinid code should be the one doing the proper checks
> its a raw mouse, this is not an acceptable solution

In practice many other types of food have "raw" tag, such as dough. After all, it's just a tag to point-out food preferences. 
From gameplay side, other races, but felinids, will keep their preferences over dead mices.

**Comment by Kush1Push1** [comment]
I actually don't see a solution without massive overhaul if "RAW" should be must-have for raw objects, even if in gameplay they keep their natural state (felinids eat dead mice, others (most) don't).

**Comment by Thunder12345** [comment]
Why would felinids eat raw mice, they're civilised people like you and me, not feral animals

**Comment by Ghilker** [comment]
> I actually don't see a solution without massive overhaul if "RAW" should be must-have for raw objects, even if in gameplay they keep their natural state (felinids eat dead mice, others (most) don't).

You'll have to do that, just removing raw won't be enough.

**Comment by Jacquerel** [comment]
I think it was me who messed this up (previously you could eat these with Deviant Tastes, now you can't), but I don't think this is an acceptable fix either.
Either Felinids are happy to eat uncooked raw food or they aren't. You would either remove RAW from the disliked food list (meaning they are happy with eating weirder things in general) or remove GORE from the liked food list again to revert a change I made (so Deviant Tastes felinids are the ones who are happy to eat dead mice and raw corpses in general).

Felinids probably already have essentially the most permissive food list in game so I don't know if making it even nicer is a good idea, I don't know how important food enjoyment balance is but it's worth mentioning.

**Comment by Draggeru** [comment]
> Why would felinids eat raw mice, they're civilised people like you and me, not feral animals

Wrong

unknown-35

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]
This is not the correct way to fix the "problem" of felinids not liking dead mice. See previous comments, although I personally prefer the callback method.

**Comment by san7890** [comment]
This has been marked as Do Not Merge for a week and has now staled out, please apply the requested changes to your PR and seek out a maintainer to re-open this.

**Pull request closed without merging**
